src/compiler/parser/index.js

```diff
@@ -528,6 +528,15 @@ function processAttrs (el) {
         name = name.replace(bindRE, '')
         value = parseFilters(value)
         isProp = false
+        if (
+          process.env.NODE_ENV !== 'production' &&
+          value.trim().length === 0
+        ) {
+          warn(
+            `${name}="${value}": ` +
+            'The v-bind expression should not be empty'
```

Comment:
I think we can improve the message to something like _the value for a v-bind expression cannot be empty. Found in '${name}'_.
Let's wait for others feedback as I am not native

Reply:
thanks, I will do it later